hack/build-bytecode-images-multi.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Set default value for IMAGE_NOO_BC tag if not already set
+IMAGE_NOO_BC=${IMAGE_NOO_BC:-quay.io/bpfman-bytecode/netobserv}
+
+# PROGRAMS is a list of <program name>:<program type> tuples
+PROGRAMS='{
+"tcx_ingress_flow_parse":"tcx",
+"tcx_egress_flow_parse":"tcx",
+"tc_ingress_flow_parse":"tc",
+"tc_egress_flow_parse":"tc",
+"tcx_ingress_pca_parse":"tcx",
+"tcx_egress_pca_parse":"tcx",
+"tc_ingress_pca_parse":"tc",
+"tc_egress_pca_parse":"tc",
+"tcp_rcv_fentry":"fentry",
+"tcp_rcv_kprobe":"kprobe",
+"kfree_skb":"tracepoint",
+"rh_network_events_monitoring":"kprobe"
+}'
+
+# MAPS is a list of <map name>:<map type> tuples
+MAPS='{
+"direct_flows":"ringbuf",
+"aggregated_flows":"per_cpu_hash",
+"packets_record":"perf_event_array",
+"dns_flows":"hash",
+"global_counters":"per_cpu_array",
+"filter_map":"lpm_trie"
+}'
+
+docker buildx create --use
```

Comment:
this fails with podman

Reply:
I think we are limited to docker for now

Reply:
(though I didn't test with bpfman)

Reply:
I sticked with docker because this is what we use to generate bytecode

Reply:
I use podman for both :-)